### PR TITLE
Bug 1210440 - Lazy load HomeView scripts

### PR DESCRIPTION
**Pull request by wilsonpage**

### CODE REVIEW

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=c93e5f09d4ddcb2c09b2240926cf210a3b98380c

File: apps/music/views/home/view.css [hunk 1]
```diff
@@ -85,4 +85,10 @@ body[data-search="true"] {
   width: 100%;
   height: 100%;
   object-fit: cover;
+  opacity: 0;
+  transition: opacity 250ms;
```
Comment (wilsonpage) [inline apps/music/views/home/view.css]: Added some polish while I was here :)

File: apps/music/views/home/view.js [hunk 2]
```diff
@@ -6,44 +6,77 @@ var HomeView = View.extend(function HomeView() {
...
+HomeView.prototype.setupPhase2 = function() {
+  return new Promise((resolve) => {
+    setTimeout(() => {
+      LazyLoader.load(this.nonCriticalScripts).then(() => {
+        this.setupSearch();
+        resolve();
+      });
+    }, 500);
+  });
+};
```
Comment (wilsonpage) [inline apps/music/views/home/view.js]: Separated setup into two 'phases': critical and non-critical.

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=3d0262bb8d7750e7b7a5f1ff9e482d0e6845bd9a

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=1562375d3a9aa0209fac76563d00b216529cea1e

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=a206cbb1fabe05cb702e00ff2fcb058988e1d2b5

**Pull request merged**